src/repart/repart.c

```diff
@@ -10065,44 +10065,48 @@ static int resize_backing_fd(
 static int determine_auto_size(
                 Context *c,
                 int level,
-                bool ignore_allocated, /* If true, determines unallocated space needed */
-                uint64_t *ret) {
+                uint64_t *ret_new_size,
+                uint64_t *ret_current_size) {
```

Comment:
Please change the order here. Since the "current size" existed before "new size", the parameters should be `ret_current_size`, `ret_new_size`.